app/heron/factories/tube_rack.rb

```diff
@@ -97,7 +97,7 @@ def create_contents!
 
       def containers_for_locations
         @tube_rack.racked_tubes.each_with_object({}) do |racked_tube, memo|
-          memo[unpad_coordinate(racked_tube.coordinate)] = racked_tube.tube
+          memo[unpad_coordinate(racked_tube.coordinate)] = racked_tube.tube.receptacle
         end
```

Comment:
Tubes and other single receptacle labware were delegating aliquots and aliquots= to the receptacle. This was only ever intended to be a temporary situation when the labware-receptacle split happened. It was disrupting eager-loading of aliquots and their tags in the API, resulting it taking ages to get a tube.

Fixing this had a handful of knock on effects where we needed to modify that aliquot association. This was one example.